### DISCUSSION TITLE
user server issue

### DISCUSSION THREAD

**Discussion by Maheshdvp999** [learndevopsonline/learndevopsonline#1345] [Q&A]
Hi team I set my frontend, mongodb, catalogue , redis and user components all are working fine but user . could u please help me on this I had checked twice .
![image](https://github.com/learndevopsonline/learndevopsonline/assets/134056076/1b7ed14d-12d2-4baf-b917-54bb14df13ea)
![Uploading image.png…]()

**Comment by Maheshdvp999** [comment]
frontend  54.83.171.160 , 172.31.45.133
mongo  35.175.254.212 , 172.31.34.55
catalogue 3.94.250.137 , 172.31.38.179
redis  54.83.153.81 , 172.31.36.56
user  3.88.5.83 , 172.31.35.156

**Reply by maheshbusireddy** [reply]
Hi @Maheshdvp999 Error in user.service script.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/c97cfc35-9acb-4d97-94be-24ba144f49ab)
line no:6 spell mistake it's not reddis it is redis.devops999.store.

**Reply by Maheshdvp999** [reply]
I had edited that spelling later pushed that code and created a new instance mahesh but its not working fine

**Reply by maheshbusireddy** [reply]
Share me new ip address

**Reply by Maheshdvp999** [reply]
User 54.234.107.129 ,172.31.42.53

**Reply by maheshbusireddy** [reply]
its working fine now. once you created new instance for user and you ran the script  for user. you need to restart the nginx server then only changes reflected at frontend server.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/46a9737a-fb6a-4ca3-9f64-ffe804e25c16)

**Reply by Maheshdvp999** [reply]
you mean I should give sudo systemctl restart nginx
